public/js/reducers/ui.js

```diff
@@ -29,7 +29,7 @@ const State = makeRecord(({
 
 function update(state = State(), action: Action) : Record<SidebarsState> {
   switch (action.type) {
-    case C.COLLAPSE_SIDEBAR: {
+    case C.TOGGLE_SIDEBAR: {
```

Comment:
We need to change all the `C.*` usages to `constants.*`. It's really important to be consistent and that's not only what we use elsewhere, but generally what many redux projects use.

I was going to look at that other PR but it got merged, otherwise I would have mentioned it there, sorry.